webv2/web.go

```diff
@@ -734,6 +734,161 @@ type DropDetail struct {
 	Name string `json:"Name"`
 }
 
+func restoreTable(w http.ResponseWriter, r *http.Request) {
```

Comment:
Foreign key id is not getting restored while restoring table

Reply:
Fixed